edu.cuny.hunter.log.tests.report/pom.xml

```diff
@@ -43,13 +43,13 @@
 		<dependency>
 			<groupId>edu.cuny.hunter.log</groupId>
 			<artifactId>edu.cuny.hunter.log.core</artifactId>
-			<version>1.0.0-SNAPSHOT</version>
+			<version>1.1.0-SNAPSHOT</version>
```

Comment:
I don't think this one had any changes. You need to issue that git command in Asana and have a look at the code that changes. Then, relate to that to the plug-ins that changed, and bump those versions. Also, we need a change summary to include in the release.

Reply:
The tests depend on the core package and it's version has changes. So, I just updated the version in the pom file

Reply:
I didn't bump up the package number for any the test package. This is only for the core which the test uses

Reply:
OK I see. It's just a dependency.